test/unit/BPool.t.sol

```diff
@@ -702,32 +702,134 @@ contract BPool_Unit_ExitPool is BasePoolTest {
   }
 
   function test_HappyPath(ExitPool_FuzzScenario memory _fuzz) public happyPath(_fuzz) {
-    bPool.exitPool(_fuzz.poolAmountIn, _zeroAmountsArray()); // Using min possible amounts
+    bPool.exitPool(_fuzz.poolAmountIn, _zeroAmountsArray());
   }
 
-  function test_Revert_NotFinalized() private view {}
+  function test_Revert_NotFinalized(ExitPool_FuzzScenario memory _fuzz) public happyPath(_fuzz) {
+    _setFinalize(false);
 
-  function test_Revert_MathApprox() private view {}
+    vm.expectRevert('ERR_NOT_FINALIZED');
+    bPool.exitPool(_fuzz.poolAmountIn, _zeroAmountsArray());
+  }
```

Comment:
we can ignore the happyPath here if i'm not mistaken, just call the pol with anything when is not finalized and it'll revert